source/Calamari/Commands/ICommandWithInputs.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Calamari.Serialization;
+using Newtonsoft.Json;
+
+namespace Calamari.Commands
+{
+    public interface ICommandWithInputs
+    {
+        int Execute(string inputs);
```

Comment:
Given all implementation here either `throw` to `return 0` it makes me wonder if this signature should return `void`?

Reply:
I agree, seems a bit pointless here.

Reply:
Fixed